lib/transaction.js

```diff
@@ -208,32 +205,32 @@ class Transaction extends EventEmitter {
   // Acquire a connection and create a disposer - either using the one passed
   // via config or getting one off the client. The disposer will be called once
   // the original promise is marked completed.
-  acquireConnection(client, config, txid) {
+  acquireConnection(config, cb) {
```

Comment:
why get rid of `txid`? it was always passed from this.txid anyway?

Reply:
Yes

Reply:
https://github.com/knex/knex/pull/3552/files#diff-e652b5ccb2708f628a9de325cf105fb7R19